### ISSUE TITLE
Conraid isn't available

### ISSUE THREAD

**Issue by nobbilie** [ich777/un-get#16]
"Can't fetch package list from repository conraid, please try again later...!"

Conraid moved to slackers.it.

**Comment by conraid** [comment]
Yes, I moved to slackers.it a few months ago. I apologize for the inconvenience. I alerted some program and repository maintainers, I did not know this.

**Comment by ich777** [comment]
@nobbilie thank you for the heads up, I will change this ASAP (few days usually).

@conraid no worries, you can't know everything. :)

**Comment by ich777** [comment]
@nobbilie I took a quick look at it and it should be as easy as editing the file `/boot/config/plugins/un-get/sources.list` on your USB Boot device and replace the line:
`https://slack.conraid.net/repository/slackware64-current/ conraid`
with
`https://slackers.it/repository/slackware64-current/ conraid`

**Comment by nobbilie** [comment]
That worked. Thank you very much!

**Comment by ich777** [comment]
I'll pushed an update to the plugin that ensures that the URL to the repository from @conraid points to the new URL.
